src/overlays/actors/ovl_Bg_Hidan_Fslift/z_bg_hidan_fslift.c

```diff
@@ -72,69 +72,76 @@ void BgHidanFslift_Destroy(Actor* thisx, PlayState* play) {
-        heightBool = false;
+        nearHomePos = false;
         if ((this->dyna.actor.world.pos.y - this->dyna.actor.home.pos.y) < 0.5f) {
-            heightBool = true;
+            nearHomePos = true;
         }
-        if (DynaPolyActor_IsPlayerAbove(&this->dyna) && heightBool) {
-            this->actionFunc = func_808870D8;
-        } else if (!heightBool) {
-            this->actionFunc = func_8088706C;
+        if (DynaPolyActor_IsPlayerAbove(&this->dyna) && nearHomePos) {
+            this->actionFunc = BgHidanFslift_Ascend;
+        } else if (!nearHomePos) {
+            this->actionFunc = BgHidanFslift_Descend;
         }
     }
 }
 
-void func_8088706C(BgHidanFslift* this, PlayState* play) {
+void BgHidanFslift_Descend(BgHidanFslift* this, PlayState* play) {
     if (Math_StepToF(&this->dyna.actor.world.pos.y, this->dyna.actor.home.pos.y, 4.0f)) {
         Actor_PlaySfx(&this->dyna.actor, NA_SE_EV_BLOCK_BOUND);
-        func_80886FB4(this);
+        BgHidanFslift_SetupWaitForPlayerOnTop(this);
     } else {
         func_8002F974(&this->dyna.actor, NA_SE_EV_ELEVATOR_MOVE3 - SFX_FLAG);
     }
-    func_80886F24(this);
+    BgHidanFslift_SetHookshotTargetPos(this);
 }
 
-void func_808870D8(BgHidanFslift* this, PlayState* play) {
+void BgHidanFslift_Ascend(BgHidanFslift* this, PlayState* play) {
     if (DynaPolyActor_IsPlayerAbove(&this->dyna)) {
         if (Math_StepToF(&this->dyna.actor.world.pos.y, this->dyna.actor.home.pos.y + 790.0f, 4.0f)) {
             Actor_PlaySfx(&this->dyna.actor, NA_SE_EV_BLOCK_BOUND);
-            func_80886FB4(this);
+            BgHidanFslift_SetupWaitForPlayerOnTop(this);
         } else {
             func_8002F974(&this->dyna.actor, NA_SE_EV_ELEVATOR_MOVE3 - SFX_FLAG);
         }
     } else {
-        func_80886FB4(this);
+        BgHidanFslift_SetupWaitForPlayerOnTop(this);
     }
-    func_80886F24(this);
+    BgHidanFslift_SetHookshotTargetPos(this);
 }
 
 void BgHidanFslift_Update(Actor* thisx, PlayState* play) {
     BgHidanFslift* this = (BgHidanFslift*)thisx;
 
     this->actionFunc(this, play);
     if (DynaPolyActor_IsPlayerOnTop(&this->dyna)) {
-        if (this->unk_16A == 0) {
-            this->unk_16A = 3;
+        if (this->cameraSetting == CAM_SET_NONE) {
+            this->cameraSetting = CAM_SET_DUNGEON0;
         }
         Camera_ChangeSetting(play->cameraPtrs[CAM_ID_MAIN], CAM_SET_ELEVATOR_PLATFORM);
     } else if (!DynaPolyActor_IsPlayerOnTop(&this->dyna)) {
-        if (this->unk_16A != 0) {
+        if (this->cameraSetting != CAM_SET_NONE) {
+            /** 
+             * It's unclear if the "cameraSetting" variable was perhaps meant to be used
+             * in this function call instead of the CAM_SET_DUNGEON0 literal. Without
+             * that, the variable's name or purpose is not obvious, so the chosen
+             * "cameraSetting" is somewhat speculative based on its role in controlling
+             * the main camera here.
+             */
```

Comment:
We reserve block comments like `/** */` for really important stuff like function level descriptions.
you can use `// `, and possibly try and be more brief. I can take a crack at it:
```
// Given the values that get set to `cameraSetting`, it seems likely that the variable was intended to be passed to the function call below. But instead it uses `CAM_SET_DUNGEON0` directly.
```

The formatter may split that into two lines, which is fine

Reply:
Oh gotcha good to know. Your version sounds good to me - updated!